lib/cpp/README.md

```diff
@@ -55,15 +54,22 @@ you are using libthriftnb you will also need libevent.
 
 ## Dependencies
 
-boost shared pointers
-http://www.boost.org/libs/smart_ptr/smart_ptr.htm
+If your C++ environment implements C++11 or later, boost will automatically use
```

Comment:
Got it, I can correct this.  I'm still dealing with aligning the stars on this one.  Lots of namespace and using cleanup, and we will no longer work with Visual Studio 2010 and 2012 in C++0x/C++11 mode; 2013 will be required for C++11 because the solution requires template aliases.